data/json/items/comestibles/drink.json

```diff
@@ -1008,6 +1008,18 @@
     "flags": [ "EATEN_HOT" ],
     "fun": 6
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "tea_willowbark",
+    "copy-from": "tea",
+    "name": { "str_sp": "willowbark tea" },
+    "healthy": -1,
+    "fatigue_mod": 0,
+    "stim": 0,
+    "description": "A bitter tea made by boiling willow bark in water, used since ancient times as a mild painkiller.",
+    "fun": -2,
+    "use_action": { "type": "consume_drug", "effects": [ { "id": "pkill1", "duration": 90 } ] }
```

Comment:
This should probably have a nausea (ideally, would be heartburn) side effect - why aspirin (acetylsalicylic acid) was invented was to try to tone down the negative stomach effects.